templates/lightbox.html

```diff
@@ -0,0 +1,58 @@
+<style>
+    .lightbox {
+        position: fixed;
+        z-index: 1000;
+        top: 0;
+        width: 100%;
+        height: 100%;
+        background-color: rgba(22,27,34,0.8);
+        display: none;
+    }
+
+    .lightbox-active {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+</style>
+<script>
+    let open_lightboxes = []
+
+    function createLightbox(id_lightbox, id_content){
+        const lightbox = document.createElement('div')
+        lightbox.id = id_lightbox
+        lightbox.classList.add('lightbox')
+        document.body.appendChild(lightbox)
+
+        const content = document.getElementById(id_content)
+        lightbox.appendChild(content)
+
+        lightbox.addEventListener('click', e => {
+            if(e.target !== e.currentTarget) return
+            lightbox.classList.remove('lightbox-active')
+            for(let i=0;i<open_lightboxes.length;i++){
+                if(open_lightboxes[i] === lightbox.id){
+                    open_lightboxes.splice(i,1)
```

Comment:
```suggestion
                    open_lightboxes.splice(i, 1)
```

nit: space after comma when separating arguments